### ISSUE TITLE
Bottom bar can't be set to a custom height

### ISSUE THREAD

**Issue by noway** [mapbox/mapbox-navigation-ios#2765]
**Mapbox Navigation SDK version:**
pod 'MapboxNavigation', '~> 1.0'

### Steps to reproduce

 1. Use 'Custom Top & Bottom Bars' example from https://github.com/mapbox/navigation-ios-examples
 2. set `let bannerHeight: CGFloat = 10.0` in https://github.com/mapbox/navigation-ios-examples/blob/main/Navigation-Examples/Examples/CustomBars/CustomBarsViewController.swift#L37
 3. Remove autoresizing mask and set custom frame and bounds here https://github.com/mapbox/navigation-ios-examples/blob/main/Navigation-Examples/Examples/CustomBars/CustomBottomBannerView.swift#L40
```swift
        contentView.frame = CGRect(x: 0, y: 0, width: 50.0, height: 10.0)
        contentView.bounds = CGRect(x: 0, y: 0, width: 50.0, height: 10.0)
//        contentView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
```

### Expected behavior
Bottom bar height to change

### Actual behavior
Bottom bar height remains the same without any seeming way to change it. 

### Justification
We used to hide bottom bar entirely and display our own on right after the navigation view as a separate view. Now it is not possible since the Resume button and user puck are always fixed at the specific bottom bar position which can't be edited in any way.

**Comment by MaximAlien** [comment]
@noway, I think similar issue was reported in https://github.com/mapbox/mapbox-navigation-ios/issues/2698. Fix was added in https://github.com/mapbox/mapbox-navigation-ios/pull/2705. It should be available in `1.2.0` release, but if you want to check whether it really fixes your case you can check-out latest pre-release tag `1.2.0-alpha.3`.

**Comment by noway** [comment]
Setting `bottomBanner.view.heightAnchor.constraint(equalToConstant: bannerHeight).isActive = true` works perfectly for me on `1.2.0-alpha.3`. Thank you! 👍👏🎉
